gs-rest-server/src/main/java/server/FixMyBugController.java

```diff
@@ -53,7 +53,7 @@ public DatabaseEntryListWrapper fixMyBug(@RequestBody String input) {
         return new DatabaseEntryListWrapper(new ArrayList<>());
     }
 
-    @RequestMapping("/test")
+    @RequestMapping("/echo")
```

Comment:
Alters the required URL for the echo method to be consistent as "echo" instead of "test"